DeGruyter.js

```diff
@@ -275,19 +277,23 @@ var testCases = [
 						"firstName": "André"
 					}
 				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
-				"attachments": [],
-				"title": "Datenbanksysteme, Eine Einführung",
-				"publisher": "De Gruyter",
-				"place": "Berlin, Boston",
-				"ISBN": "978-3-486-59834-6",
-				"url": "http://www.degruyter.com/view/product/218879",
+				"date": "2015",
+				"ISBN": "9783110443752",
+				"abstractNote": "This textbook provides a systematic and thorough introduction to modern database systems. It presents today's leading relational technology in special detail. The 10th edition updates and expands the presentation of recent developments (i.e. main memory database systems and Big Data applications). Extensive examples illustrate the concepts, making the book especially suitable for self study.",
+				"extra": "10th expanded and updated edition",
```

Comment:
but this can be other things? We can't just put it into the edition field? If so, I'm wondering whether it doesn't make more sense to push it to a note. I think that's more in line with what we do e.g. in library catalogs.